src/main/java/utils/CustomStringUtils.java

```diff
@@ -0,0 +1,43 @@
+package utils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CustomStringUtils {
+    private static final String DEFAULT_DELIMITER_TEXT = ",|:";
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final String ERROR_MESSAGE_INVALID_NEGATIVE_NUMBER = "[Error] 음수는 입력할 수 없습니다.";
+    private static final String ERROR_MESSAGE_INVALID_NUMBER_FORMAT = "[Error] 올바른 숫자가 아닙니다.";
+
+    public static boolean isNullOrEmpty(String str) {
+        if (str == null || str.isEmpty()) {
+            return true;
+        }
+        return false;
```

Comment:
한 줄로 표현하셔도 충분할 것 같습니다 😄 
```suggestion
        return (str == null || str.isEmpty());
```

Reply:
앗 그러네요^^ 의견 감사합니다.